tests/integration-tests/tests/common/scaling_common.py

```diff
@@ -156,3 +160,14 @@ def get_batch_ce_min_size(stack_name, region):
         .get("computeResources")
         .get("minvCpus")
     )
+
+
+def emulate_ice_in_cluster(cluster):
```

Comment:
is this the setup of the override for EC2 launch API calls, so to emulate ICE? Can we find a better naming, so that is clear that it's not emulating any EC2 launch instance call, but only setting up things?

Reply:
e.g. `setup_ec2_launch_override_to_emulate_ice` ?